src/interactive/documentation.ts

```diff
@@ -2,6 +2,30 @@ import * as path from 'path'
 import * as vscode from 'vscode'
 import { withLanguageClient } from '../extension'
 import { getVersionedParamsAtPosition, setContext } from '../utils'
+import MarkdownIt = require('markdown-it')
```

Comment:
This looks weird. Should either use `require` or `import`.

Reply:
mostly addressed in #1456 , which for `@traptitech/markdown-it-katex`, we can't just use `import` because of its weird typing: https://github.com/traPtitech/markdown-it-katex/blob/master/index.d.ts